tests/src/Unit/OgAccessEntityTestBase.php

```diff
@@ -42,19 +43,32 @@ public function setup() {
     $entity_type->isSubclassOf(FieldableEntityInterface::class)->willReturn(TRUE);
     $entity_type->id()->willReturn($entity_type_id);
 
+    // Some mocked groups that are referenced by the content entity.
+    $groups = [
+      $this->prophesize(ContentEntityInterface::class),
+      $this->prophesize(ContentEntityInterface::class),
+    ];
+
+    $og_audience = $this->prophesize(EntityReferenceFieldItemList::class);
+    $og_audience->referencedEntities()->willReturn($groups);
+
     $this->groupContentEntity = $this->prophesize(ContentEntityInterface::class);
     $this->groupContentEntity->id()->willReturn($entity_id);
     $this->groupContentEntity->bundle()->willReturn($bundle);
     $this->groupContentEntity->isNew()->willReturn(FALSE);
     $this->groupContentEntity->getEntityType()->willReturn($entity_type->reveal());
     $this->groupContentEntity->getEntityTypeId()->willReturn($entity_type_id);
+    $this->groupContentEntity->get('og_audience')->willReturn($og_audience);
```

Comment:
This og_audience can be replaced by `OgGroupAudienceHelperInterface::DEFAULT_FIELD`